packages/cloud-cognitive/src/components/UserProfileImage/UserProfileImage.js

```diff
@@ -142,9 +143,9 @@ UserProfileImage.propTypes = {
    */
   className: PropTypes.string,
   /**
-   * When passing the icon prop, use either "user" or "group". The values match up to the Carbon Library icons.
+   * When passing the kind prop, use either "user" or "group". The values match up to the Carbon Library icons.
    */
-  icon: PropTypes.oneOf(['user', 'group']),
+  kind: PropTypes.oneOf(['user', 'group']),
```

Comment:
Per the check, let's go ahead and sort the prop types alphabetically so this darn thing will pass 😅